config/channels/in-memory-channel/webhooks/resource-validation.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The Knative Authors
```

Comment:
I'm thinking since you're touching the files, have all licenses of those updated?